.coveragerc

```diff
@@ -1,12 +1,11 @@
 [run]
-branch = True
```

Comment:
Okay for 100 % line coverage even if I'd prefer 100 % branch coverage. 

(In a personal project I switched to branch coverage and it was an eye opener showing edge cases I did not think of before. But it was hard to gain 100 % branch coverage there. I had to trick `coverage.py` multiple times by using constructs which are not handled as branches.)

But I do not have the time to invest into perfect branch coverage here, so I am happy that you brought it to 100 % line coverage.